nat-lab/tests/utils/ping.py

```diff
@@ -12,6 +12,13 @@
 # It should work for Linux, Windows and Mac.
 
 
+async def ping(
+    connection: Connection, ip: str, timeout: Optional[float] = None
+) -> None:
+    async with Ping(connection, ip).run() as ping:
+        await ping.wait_for_any_ping()
+
+
 class Ping:
```

Comment:
nit: Could we move whole `Ping` class definition inside of the `ping` function? This way it would be impossible to use by accident.

Reply:
I have left the class exposed, as conntracker is using "long-running" pings. But instead of modifying `wait_for_next_ping()` I have created new method called `wait_for_any_ping()`. 

This way we can still have this long-running pings if needed exposed via `Ping` class, but for most use-cases simply using `ping()` will be sufficient function.